dashboard/app/views/levels/_external_link.html.haml

```diff
@@ -4,4 +4,4 @@
 
   %h4= link_to @level.link_title, @level.url, target: '_blank'
 
-  %div{style: 'padding-top: 100px'}!= t('external_links.disclaimer_md')
+  %div{style: 'padding-top: 100px'}!= t('external_links.disclaimer_md', support_url: t('landing.support_url'))
```

Comment:
This should just be the url directly `support_url: https://support.code.org`. We shouldn't be translating the URL, on my plate to remove existing places using this.